src/utility/idb.js

```diff
@@ -0,0 +1,56 @@
+import { createStore } from "idb-keyval";
+import {
+  get,
+  set,
+  getMany,
+  setMany,
+  del,
+  clear,
+  entries,
+  keys,
+  values
+} from "idb-keyval";
+
+const DB = "zesty";
+const STORE = "manager-ui";
+
+// use a custom store so we can recreate the reference
+// if it gets deleted during runtime
+let customStore = createStore(DB, STORE);
+
+export default {
```

Comment:
I feel like there is a terser way to do this instead of re-implementing every function. Maybe proxies but that looks like it'll end up being the same if not more code. Some sort of dynamic function call reference.